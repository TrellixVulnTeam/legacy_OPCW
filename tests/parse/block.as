# Some block statements
{x;}
{+30;}
{}

# Some block expressions
{+{30;};}
