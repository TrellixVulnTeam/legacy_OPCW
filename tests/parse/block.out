ModuleDecl <?> _
  Nodes <?> 
    Block <?>
      Nodes <?> 
        Ident <?> x
    Block <?>
      Nodes <?> 
        NumericPromoteExpr <?>
          IntegerExpr <?> 30 (10)
    Block <?>
      Nodes <?> 
    Block <?>
      Nodes <?> 
        NumericPromoteExpr <?>
          Block <?>
            Nodes <?> 
              IntegerExpr <?> 30 (10)
